qemu.mk

```diff
@@ -41,13 +41,15 @@ define bios-qemu-common
 	+$(MAKE) -C $(BIOS_QEMU_PATH) \
 		CROSS_COMPILE=$(CROSS_COMPILE_NS_USER) \
 		O=$(ROOT)/out/bios-qemu \
-		BIOS_NSEC_BLOB=$(LINUX_PATH)/arch/arm/boot/zImage \
-		BIOS_NSEC_ROOTFS=$(GEN_ROOTFS_PATH)/filesystem.cpio.gz \
-		BIOS_SECURE_BLOB=$(OPTEE_OS_BIN) \
 		PLATFORM_FLAVOR=virt
 endef
 
 bios-qemu: update_rootfs optee-os
+	ln -sf $(OPTEE_OS_HEADER_V2_BIN) .
+	ln -sf $(OPTEE_OS_PAGER_V2_BIN) .
+	ln -sf $(OPTEE_OS_PAGEABLE_V2_BIN) .
+	ln -sf $(LINUX_PATH)/arch/arm/boot/zImage .
+	ln -sf $(GEN_ROOTFS_PATH)/filesystem.cpio.gz rootfs.cpio.gz
```

Comment:
This fills the **build/** directory with the optee images.
Could we do as the qemu/v8, store bins in a `$(BIOS_QEMU_PATH)/build/release` and run qemu for the later?

Reply:
Except that we don't have any such path here as we're not building ARM-TF.
How about putting it in `out/` or `out/bin` instead?

Reply:
out/ or out/bin are fine. Thanks.